lib/__tests__/created_message_queue_handler.test.ts

```diff
@@ -6,6 +6,7 @@
 // tslint:disable-next-line:no-object-mutation
 process.env = {
   ...process.env,
+  API_PROXY_WEBHOOK_URL: "http://aUrl.com",
```

Comment:
using `API_PROXY` in this context could be confusing, I would just use something like `WEBHOOK_CHANNEL_URL`

Reply:
done